src/view.js

```diff
@@ -105,7 +107,8 @@ var View = Backbone.View.extend({
 });
 
 _.extend(View.prototype, ViewMixin);
-
 _.extend(View.prototype, RegionsMixin);
+_.extend(View.prototype, BehaviorsMixin);
```

Comment:
So I initially was all for listing every mixin.. but I think I'm swinging back the other way.  ViewMixin will not work without BehaviorsMixin and UIMixin.. so I think they should be mixed into it, and they don't need to be here.  ViewMixin may have more stuff taken out of it, but it is a mixin that composes others.  And that doesn't keep us from using the mixins by themselves like we're doing with UIMixin on behavior.